### DISCUSSION TITLE
Paste command

### DISCUSSION THREAD

**Discussion by JoKo1959** [chrisgurney/obsidian-note-toolbar#166] [Scripting]
**Description of the bug**
On iOS Obsidian mobile, the paste-command is available via the built-in toolbar. Now I would like to have the paste-command available via the toolbar of the Note Toolbar plugin. But this is not possible. If I open the edit mode of an item and enter only the first two letters of “Paste” in the search for command field, i.e. “Pa”, the paste command is displayed in the pop-up window, but I cannot select it; the message “This command does not exist” appears. What solution is there for this? Thank you very much!

**Steps to reproduce**
see above

**Mobile environment (if applicable to the issue):**
 - Device: iPhone 15 Pro max
 - OS: 18.1
 - Obsidian version: 1.7.4
 - Note Toolbar version: 1.14.9

**Comment by chrisgurney** [comment]
Thanks for logging this @JoKo1959 ! 

Unfortunately Obsidian doesn't expose several tools in the editor's toolbar as commands. This means they would have to be added to Note Toolbar directly, another plugin might expose these commands, or they can be handled with scripting support (currently [in beta](https://github.com/chrisgurney/obsidian-note-toolbar/discussions/171)).

I'm going to move this over to the discussion forum, as there are several tools that might be added. In the back of my mind I'm trying to keep bloat on the plugin down, but may consider some of these commands if there's enough demand.

**Comment by chrisgurney** [comment]
I've added an example of a [formatting toolbar](https://github.com/chrisgurney/obsidian-note-toolbar/wiki/Examples#formatting) that includes a `paste` item. A script is required, so make sure to get whichever works for you from the `examples` folder (in the repo) which is linked from that page.

**Comment by chrisgurney** [comment]
@JoKo1959 
_Paste_, as well as a number of other items are available in the Note Toolbar Gallery (as of v1.21).
